### DISCUSSION TITLE
Allow validation on nested relational values through parent validation settings

### DISCUSSION THREAD

**Discussion by ananth3087** [directus/directus#24925] [Feature Requests]
### Describe the Bug

When we configure the validation for child data model from parent data model using validation tab and then its not working in latest version ( 9.21.2), but it was working in old version like 9.10.0. 

Note: I dont want to configure the validation in Child data modal because i planned to create multiple relationship field with same Child data modal, so each time, i will use different validation rules, so Parent is the better place to configure it. 

- I have created two entity such as Parent and Child. 
- I have related the Parent and child using Many to many relationship. 

Parent Entity with Many to many relationship with child: 

<img width="1430" alt="image" src="https://user-images.githubusercontent.com/38585821/206251583-464125fb-1681-419b-823f-8a87a9c83f66.png">

Child Entity: 

<img width="1389" alt="image" src="https://user-images.githubusercontent.com/38585821/206253412-f549e593-8250-477f-81bd-1d56add03b95.png">

In parent, we child field (many to many field), and then there is a validation tab in which i have configure the validation for field in child, but its not working.

<img width="1429" alt="image" src="https://user-images.githubusercontent.com/38585821/206254156-7df5a867-f673-45de-9b5b-61abde3b9f17.png">

In following screenshot, you can see that i'm able to create child with empty title

<img width="1441" alt="image" src="https://user-images.githubusercontent.com/38585821/206259549-b6b34b20-8a27-49fa-bb36-9d30b0939e0a.png">

 
<img width="1441" alt="image" src="https://user-images.githubusercontent.com/38585821/206259678-f46ff227-59bb-48a0-9c17-d08fb7568f3b.png">

### To Reproduce

We can use latest version above 9.12 to reproduce this issue. 

- I have created two entity such as Parent and Child. 
- I have related the Parent and child using Many to many relationship. 

Parent Entity with Many to many relationship with child: 

<img width="1430" alt="image" src="https://user-images.githubusercontent.com/38585821/206251583-464125fb-1681-419b-823f-8a87a9c83f66.png">

Child Entity: 

<img width="1389" alt="image" src="https://user-images.githubusercontent.com/38585821/206253412-f549e593-8250-477f-81bd-1d56add03b95.png">

In parent, we child field (many to many field), and then there is a validation tab in which i have configure the validation for field in child, but its not working.

<img width="1429" alt="image" src="https://user-images.githubusercontent.com/38585821/206254156-7df5a867-f673-45de-9b5b-61abde3b9f17.png">

In following screenshot, you can see that i'm able to create child with empty title

 
<img width="1441" alt="image" src="https://user-images.githubusercontent.com/38585821/206259549-b6b34b20-8a27-49fa-bb36-9d30b0939e0a.png">

 
<img width="1441" alt="image" src="https://user-images.githubusercontent.com/38585821/206259678-f46ff227-59bb-48a0-9c17-d08fb7568f3b.png">

### What version of Directus are you using?

9.21.2

### What version of Node.js are you using?

18.12

### What database are you using?

sQLite

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

**Comment by ghost** [comment]
Same here. Tried to validate files’ mime-type to only allow images … not possible :(
